test/xpu_api/numerics/complex.number/complex.member.ops/divide_equal_complex.pass.cpp

```diff
@@ -49,7 +49,9 @@ ONEDPL_TEST_NUM_MAIN
 {
     test<float>();
     IF_DOUBLE_SUPPORT(test<double>())
+#if !_PSTL_ICC_TEST_COMPLEX_NON_FLOAT_COMPLEX_DEPRECATED
     IF_LONG_DOUBLE_SUPPORT(test<long double>())
+#endif // !_PSTL_ICC_TEST_COMPLEX_NON_FLOAT_COMPLEX_DEPRECATED
```

Comment:
Isn't `long double` one of the still supported types?  (seems to be done in 4 places)

Reply:
Good point, fixed, thanks!